.github/workflows/docs.yml

```
@@ -17,42 +17,44 @@ jobs:
           name: docs-html
           path: docs/_build/html/
-  
+
   apidocs:
     if: github.event.pull_request.merged != true || github.ref != 'refs/heads/main'
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      
+
       - uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)